component/phlare/push.v1.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package push.v1;
```

Comment:
We need to try to wire this up into `go generate` and our build image so we can regenerate the files as needed.

Reply:
Yeah I think we can create an issue for this. Phlare use buff but I don't want to force anything, this was just to get a first version.

Reply:
After thinking a bit more I'm planing to remove this from here.

Reply:
This is removed.